keyboards/doro67/regular/config.h

```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define VENDOR_ID       0x4250 // BP for Backprop
+#define PRODUCT_ID      0x5247 // RG for Regular
```

Comment:
```suggestion
#define PRODUCT_ID      0x5243 // RE for Regular
```
?

Reply:
```suggestion
#define PRODUCT_ID      0x5245 // RE for Regular
```

E is `45`, not `43`.

Reply:
🤦‍♂